logging/google/cloud/logging/handlers/container_engine.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Logging handler for Google Container Engine (GKE).
+
+Formats log messages in a JSON format, so that Kubernetes clusters with the
+fluentd google cloud plugin installed can format their log messages so that
+metadata such as log level is properly captured.
+"""
+
+import json
+import logging.handlers
+import math
+
+
+class ContainerEngineHandler(logging.StreamHandler):
+    """Handler to formats log messages in good fluentd GKE format.
+
+    This handler is written to format messages for the Google Container Engine
+    (GKE) fluentd plugin, so that metadata such as log level are properly set.
+    """
+
+    def format(self, record):
+        """Format the message into JSON expected by fluentd.
+
+        :type record: :class:`~logging.LogRecord`
+        :param record: the log record
+
+        :rtype: str
+        :returns: A JSON string formatted for GKE fluentd.
+        """
+        message = super(ContainerEngineHandler, self).format(record)
+
+        subsecond, second = math.modf(
+            record.created)
+
+        payload = {
+            'message': message,
+            'timestamp': {'seconds': int(second),
+                          'nanos': int(subsecond * 1e9)},
```

Comment:
1. This is identical to `AppEngineHandler.format`. Why not share code between them?
1. Any reason this is formatted differently than the identical code in that block?